prewl/prompts.py

```diff
@@ -29,14 +29,14 @@ def validate(self):
 
         # Inputs and output are distinct and equal to keys
         assert self.output not in self.inputs, "Output can't be an input"
-        assert set(self.inputs + [self.output]) == keys, "Inputs and output should equal the keys"
+        # assert set(self.inputs + [self.output]) == keys, "Inputs and output should equal the keys"
```

Comment:
A lot of the assertions in PREWL's validation had inbuilt assumptions for the text generation task, so I commented them out for now but depending on future directions for PREWL, making validation specific to the models may be needed.

Reply:
We're doing some pretty brittle string manipulation, which makes the assertions useful. Any chance they could be extended to only fire when doing completion?

Reply:
I made it so they will run always but added and 'or' clause for if its mask filling.